website/docs/r/appsync_graphql_api.html.markdown

```diff
@@ -143,7 +165,7 @@ The following arguments are supported:
 
 The following arguments are supported:
 
-* `default_action` - (Required) The action that you want your GraphQL API to take when a request that uses Amazon Cognito User Pool authentication doesn't match the Amazon Cognito User Pool configuration. Valid: `ALLOW` and `DENY`
+* `default_action` - (Required only if Cognito is used as the default auth provider) The action that you want your GraphQL API to take when a request that uses Amazon Cognito User Pool authentication doesn't match the Amazon Cognito User Pool configuration. Valid: `ALLOW` and `DENY`
```

Comment:
this particular bit probably needs better wording - the `default_action` parameter isn't supported in the additional_auth_provider's user_pool_config block, but I wasn't sure if this is the better way, or if I should create an entirely separate user_pool_config block specifically for the additional_authentication_provider.